Automattic/jetpack
# Verbum: change reply to comment where needed

This PR fixes the wording around comments.
**Comment**
1. When posting a comment the submit button should be `Comment`.
2. The textarea placeholder should be `Write a comment...`.
3. Depending on the comment auth config we should either display: `Log in to leave a comment.` or `Leave a comment. (log in optional)`
4. The title above the form should say: `Leave a Comment`

**Reply**
1. When posting a reply the submit button should be `Reply`.
2. The textarea placeholder should be `Write a reply...`.
3. Depending on the comment auth config we should either display: `Log in to leave a reply.` or `Leave a reply. (log in optional)`

This PR fixes the e2e tests, and adds a command to run them `e2e-tests`.

Fixes #
136-gh-Automattic/verbum

## Proposed changes:

* Introduce `commentParent` state to keep track if we are replying or just commenting.
* Introduce new hook to watch for form mutations and update the new state.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout this PR
* Run the e2e tests and ensure that everything is ok
* Verify all the string changes in the PR description
